### ISSUE TITLE
Change Mito Haplogroup to 2 drop downs and alphanumeric field

### ISSUE THREAD

**Issue by clegarde** [spawaskar-cora/cora-docs#243]
I think it would make more sense to change our current mito haplogroup field to two separate drop downs and an alphanumeric field because it does not accurately reflect the haplogroup data we get. We often get something like this: H2b1a - European. The first letter is the general group, so I think it might make more sense to separate this from the ancestral group. Possibly as follows:
**Mito Haplogroup** (drop down): H, J, I, etc.
**Subgroup** (alphanumeric field): 2b1a
**Population** (drop down): European, Native American, African, Asian (separate into Southeast Asian, etc?), Other
![screen shot 2018-07-31 at 4 13 33 pm](https://user-images.githubusercontent.com/7350474/43487830-680d709a-94dd-11e8-8d28-2adf0a02bc95.png)

I briefly mentioned this to @SachinPawaskarUNO and @nmcelroy.

**Comment by emilystreetman** [comment]
Per scrum discussion, @SachinPawaskarUNO agrees to add "*Mito Haplogroup*" and "*Mito Subgroup*" as suggested by @clegarde, and *Population* would then auto-populate based on the Mito Subgroup entered (it would not be a user-entered field via a dropdown menu). 

Next step: DNA folks would need to agree on the list of options for each of these two categories and what would display in Populations for each group/subgroup

**Comment by fedamann** [comment]
@fedamann call AFDIL ET and figure out what the best way to record this data.

**Comment by clegarde** [comment]
Here is haplogroup data that the USS OK project has received from AFDIL. Other projects that have additional Hg data can add to this spreadsheet and re-post, if possible. Thanks! @TarawaProject @CTegtmeyer @kristengrow @Megyesimm @skorps @mei215 
[HaplogroupData.xlsx](https://github.com/spawaskar-cora/cora-docs/files/3109759/HaplogroupData.xlsx)

**Comment by SachinPawaskarUNO** [comment]
@clegarde Thank you for getting this started. Just to clarify that this data is for Mito Haplogroup. There will be a separate dataset/excel file for Y-Haplogroup, correct?

**Comment by clegarde** [comment]
Yes, this is mito. Anyone with Y Hg data can just add a separate tab to the spreadsheet (or upload a separate) with the same format. I forgot to add that. USS OK doesn't have any Y Hg data.

**Comment by Megyesimm** [comment]
Do you want us to just add them to the list, or attempt to incorporate them into the beautiful order that USS OK has provided? (i.e alphabetical order)   Also: a few of my Haplogroups are already captured in this list.

**Comment by SachinPawaskarUNO** [comment]
@Megyesimm Good question. No need for it to be in alphabetical order, just get all your Haplogroups in there. The CoRA system will take care of ordering them. Please do not add duplicates.

**Comment by btrnew** [comment]
Here are the haplogroup data for Tarawa. All duplicates have been avoided and unique haplogroups have been added to Carrie's list. 
[CoRAHaplogroupData.xlsx](https://github.com/spawaskar-cora/cora-docs/files/3114461/CoRAHaplogroupData.xlsx)

**Comment by btrnew** [comment]
@Megyesimm couldn't attach her updated spreadsheet for some reason, so she emailed it to me. This version has all of the Tarawa and Cab haplogroups added

[CoRAHaplogroupData1.xlsx](https://github.com/spawaskar-cora/cora-docs/files/3114608/CoRAHaplogroupData1.xlsx)

**Comment by aavorthmann** [comment]
[CoRAHaplogroupData1 - Edits.xlsx](https://github.com/spawaskar-cora/cora-docs/files/3209228/CoRAHaplogroupData1.-.Edits.xlsx)

I made a few notes regarding duplicates and an irregular ancestry listing. I have not added the noted haplogroups to the Seeder at this time. 

@clegarde @Megyesimm

**Comment by clegarde** [comment]
There is currently an extensive list of haplogroups in CoRA. I believe this can be closed.
